ask-sdk-core/ask_sdk_core/skill.py

```diff
@@ -23,6 +23,7 @@
 from ask_sdk_runtime.skill import AbstractSkill, RuntimeConfiguration
 from ask_sdk_runtime.dispatch import GenericRequestDispatcher
 from ask_sdk_runtime.exceptions import AskSdkException
+from ask_sdk_runtime.utils import UserAgentManager
```

Comment:
one question related to python. Since the UserAgentManger is defined in runtime. For skill only depend on sdk core, could it import from runtime as well ? I just realize the node implementation forget to export UserAgentManger from core thus skill only depend on core can't access UserAgentManger.

Reply:
Yes, since `runtime` is a dependency for `core`, it will be available for skill developers when only `core` is added as dependency.